src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/model/response/StaffNameResponse.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.model.response
+
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.Name
+
+data class StaffNameResponse(
+  val id: Long,
+  val name: Name,
+  val code: String,
+  val username: String?,
+)
```

Comment:
this is from project create-and-vary-a-licence-and-delius, in CVL we mixed up this with the Staff (USER in CVL) object

data class StaffName(
    val id: Long,
    val name: Name,
    val code: String,
    val username: String?
)